drivers/console/Kconfig

```diff
@@ -143,6 +143,21 @@ config RTT_CONSOLE
 	  Segger J-Link software and displayed on a computer in real-time.
 	  Requires support for Segger J-Link on the companion IC onboard.
 
+config RTT_TX_RETRY_CNT
```

Comment:
The Host check RTT every dozens of milliseconds (100ms pool rate was frequently observed). The proposed values will not work in such case (algorithm will not detect host). IMHO 10 attempts with 10ms delays between them should catch all the possible cases. @Qbicz: What do you think?

Reply:
I've seen 1-2ms period. 10ms of sleep is quite long (i don't know what is the context of this call).

Reply:
If user is only using RTT up channels (logging from device to PC), the period is usually 1-2 ms as @nordic-krch mentioned. Longer periods may appear if there are frequent RTT writes from PC to device concurrent to RTT reads. But in such case, user will probably easily observe the issue and change this value. For default, I would suggest 2 ms delay x 5 times.

Reply:
Even during bidirectional usage: logger + console, the polling rate should not exceed few milliseconds. In this scenario, characters typed by user into console (RTT writes PC > device) are not appearing very often and they do not interfere with logger (RTT reads device > PC). The delays in hundreds of milliseconds were only observed when sending larger amounts of data in both RTT directions (tens of kB/s).

Reply:
@Qbicz, @nordic-krch I see that it is not ultra simply to chose appropriate value then. Can you prepare better helper message which explain ho w to do so then? I expect something like - if you observe that expected messages are lost then increase the retries count... This remarks touch also RTT_TX_RETRY_DELAY_MS.